### PR TITLE
Added secondary capture support and documentation

### PR DESCRIPTION
**Pull request by mats-claassen**

### CODE REVIEW

File: gateway.py [hunk 1]
```diff
@@ -54,7 +54,11 @@ def __call__(self, json_response, binary_components):
             except StopIteration:
                 logger.error('Ran out of binary components for JSON part {}'.format(i))
             
-            if binary_type in {'probability_mask'}:
+            if binary_type in {'dicom_secondary_capture'}:
+                # Binary blob is assumed to be a file pointer or buffer type
+                # to be read directly into the response
+                yield ('application/dicom', binary_blob)
```
Comment (mats-claassen) [inline gateway.py:60]: I am not sure about this. What do you think @bsblaw, is `application/dicom` correct?

Comment (bsblaw) [reply gateway.py:60]: application/dicom is correct. That's what we use in in our multiple request as well.

File: README.md [hunk 2]
```diff
@@ -323,3 +324,25 @@ You must have Python 3.6+ installed.
...
+If your model's output is a secondary capture DICOM file and you want to return that as result then you have to specify `"binary_type": "dicom_secondary_capture"` in the corresponding `parts` object. Your response could look like this:
+
+```json
+{
+  "protocol_version": "1.0",
+  "parts": [
+      {
+          "label": "Mock seg",
+          "binary_type": "dicom_secondary_capture",
+          "binary_data_shape": {
```
Comment (bsblaw) [inline README.md:339]: Is the `binary_data_shape` here necessary? Perhaps the metadata is already in the dicom file?

Comment (mats-claassen) [reply README.md:339]: Yeah that is correct. This attribute could be removed

**Pull request merged**